pybleau/app/ui/dataframe_plot_manager_view.py

```diff
@@ -223,9 +223,10 @@ def _new_plot_button_fired(self):
             return
 
         plot_type = selector.plot_type
-
+        initial_creation = True
         if plot_type in list(self.model.custom_configs.keys()):
```

Comment:
Unrelated: no need for this `list` call:  x in d.keys() works as expected.

Reply:
Fixing in a separate PR.